topics/userscript/index.md

```diff
@@ -0,0 +1,12 @@
+---
+aliases: userjs, greasemonkey, tampermonkey, violentmonkey
+created_by: Aaron Boodman
+display_name: Userscripts
+github_url: https://github.com/greasemonkey/greasemonkey
+released: March 28, 2005
+short_description: A kind of lightweight browser extension, used to modify webpages.
+topic: userscript
+wikipedia_url: https://en.wikipedia.org/wiki/Userscript
+related: WebExtension, JavaScript
```

Comment:
Could you lowercase these? https://github.com/topics/webextension and https://github.com/topics/javascript are the two topics.

Reply:
done